packages/application-components/src/components/tab-header/tab.styles.ts

```diff
@@ -63,7 +63,7 @@ export const getLinkStyles = (
       :hover,
       :focus,
       :active {
-        > * > * {
+        > * {
```

Comment:
nitpick: perhaps we can take this opportunity to be more specific with this rule.
Since we now we are using an `h4` element within the link (and that's the element we want to address the color rule to), we could update it to be something like this:
```
& h4 {
```